fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1574,6 +1578,14 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VarAttr(name = ENABLE_PLAN_ADVISOR)
     private boolean enablePlanAdvisor = true;
 
+    @VarAttr(name = COUNT_DISTINCT_IMPLEMENTATION)
+    private String countDistinctImplementation = "default";
+
+    // By default, we always use the created mv's bitmap/hll to rewrite count distinct, but result is not
+    // exactly matched with the original result.
+    // If we want to get the exactly matched result, we can disable this.
```

Comment:
bitmap is exactly, you can always use bitmap to replace `count(distinct)`,

Reply:
- `bitmap_uniont(to_bitmap(x)) ` if x is numeric, it's exact with `count(disitnct); but if x is string, the result is wrong.
- mv has supported `bitmap_union(bitmap_hash(x))` to `count(distinct)`, it's also inexact.